lib/pushmi_pullyu/cli.rb

```diff
@@ -26,12 +27,19 @@ def parse(argv = ARGV)
   end
 
   def run
-    if options[:daemonize]
-      start_server_as_daemon
-    else
-      # If we're running in the foreground sync the output.
-      $stdout.sync = $stderr.sync = true
-      start_server
+    configure_rollbar
+    begin
+      if options[:daemonize]
+        start_server_as_daemon
+      else
+        # If we're running in the foreground sync the output.
+        $stdout.sync = $stderr.sync = true
+        start_server
+      end
+    # rubocop:disable Lint/RescueException
```

Comment:
Yeah rubocop is a bit silly here. Basically it should only say that if you didnt reraise the exception.

``` # Never rescue Exception *unless* you re-raise in rescue body```